cinema/models.py

```diff
@@ -95,9 +106,9 @@ def validate_ticket(row, seat, cinema_hall, error_to_raise):
                 raise error_to_raise(
                     {
                         ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
+                                          f"number must be in available range:"
+                                          f"(1, {cinema_hall_attr_name}): "
+                                          f"(1, {count_attrs})"
```

Comment:
The error message formatting in the `validate_ticket` method could be improved for clarity. Consider using a single f-string for better readability and to avoid breaking the string into multiple lines.